tests/email/test_dm_mandrill.py

```diff
@@ -139,8 +138,8 @@ def test_calls_send_email_with_alternative_reply_to(email_app, mandrill):
 
 def test_should_throw_exception_if_mandrill_fails(email_app, mandrill):
     with email_app.app_context():
-
-        mandrill.messages.send.side_effect = Error('this is an error')
+        something_bad = Exception('this is an error')
```

Comment:
Might be nice to parametrize this test to check for `JSONDecodeError` (as well as the generic `Exception`), as this is what we know we want to catch in this instance.

Reply:
Yep, might look a bit weird because everyone looking at it will know that if the behaviour is as expected for Exception, it'll also be so for JSONDecodeError, but if you think it makes things more clear then happy to.